lessons/Early-Math/commutative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,120 @@
+
+== Tying it All Together
+
+=== Overview
+
+Students connect and apply what they have learned so far about translation, computation, and the Commutative Property.
+
+=== Launch
+
+Explain to your students that Claire and Walker are both translating expressions in words into Circles of Evaluation. Their teacher asks them to represent the following expression: "one half the product of five and six."
+
+[.embedded, cols="^.^1,^.^1", grid="none", stripes="none" frame="none"]
+|===
+
+| Claire draws this:				| Walker draws this:
+|@show{(coe  '(* 1/2 (* 5 6)))}		| @show{(coe  '(* (* 5 6) 1/2))}
+|===
+
+Both students compute that this expression evaluates to 15.
+
+The teacher looks at their work. She definitely likes Claire’s Circle, but she can’t decide Walker's is a good translation or not.
+
+[.lesson-instruction]
+- What do you think? Are Claire _and_ Walker correct - or just Claire?
+** _Arguments against Walker's Circle: The expression states that we start with one half, and then multiply that by 5x6. Walker’s Circle is not a direct translation._
+** _Arguments for Walker's Circle: When we use computation, both Circles evaluate to 15. Both Walker and Claire are taking half of 30. The Commutative Property indicates that we can multiply factors in any order._
+
+Be sure to challenge students to consider both perspectives: although Claire’s Circle is a more direct translation, Walker has demonstrated (1) an understanding of the Commutative Property, and (2) an understanding of how computation can produce equivalent Circles.
+
+=== Investigate
+
+Before moving on to the next activity, emphasize to students that Walker and Claire’s Circles are _different_ but still _equivalent_. Computation helps us to verify that!
+
+[.lesson-instruction]
+- Look at @printable-exercise{claire-and-walker.adoc} with some additional work by Claire and Walker. Their teacher awards credit when her students translate the expression precisely __or__ when they show a deep understanding of computation or commutativity.
+- In the column on the right, record if Claire, Walker, or both students correctly translated the words into a Circle.
+- Optional: Complete @printable-exercise{words-to-circles-challenge.adoc} to analyze Circles of Evaluation and commutativity for a more complex expression in words.
+
+Discuss and debrief with students. Invite students to verbally share their responses to reinforce important vocabulary and concepts that students will use again and again in future lessons.
+
+=== Synthesize
+
+Do you translate words into Circles precisely and directly – or do you sometimes make changes to your Circles as you translate? Why?
+
+If you were a teacher, would you require _exact_ translations of circles, or would you allow students to modify their Circles when translating?
```

Comment:
I am curious why this is the most important point you want to make in a lesson that is introducing the commutative property.  Some teachers will feel very strongly about landing on one side or another of this conversation. I think there are probably many other ideas that all teachers and students could agree are important takeaways from this lesson...